server/api-manager.js

```diff
@@ -1063,10 +1063,24 @@ class MindbodyQueries {
       this.password = secrets.password;
       this.apikey = secrets.apikey;
     } catch (err) {
-      console.log("Error:", err);
+      console.log("Error in api-manager.loadConfig: " + err);
     }
 
   }
+
+  logNumRequests() {
+    let message = "In this entire Mindbody Onion session, " + this.requestNum + " API requests of " +
+      MAX_QUERIES + " have been used. ";
+    const numFreeCalls = 1000;
+    if (MAX_QUERIES < numFreeCalls) {
+      message = message + (MAX_QUERIES - this.requestNum) + " free daily requests remain.";
+    } else {
+      const minBilledCalls = Math.max(0, this.requestNum - numFreeCalls);
```

Comment:
This code path never runs when MAX_QUERIES is < 1000, so probably it will never run for Shannon.. But it's been very useful for me in debugging when I set MAX_QUERIES > 1000